public/views/modals/customized-amount.html

```diff
@@ -0,0 +1,80 @@
+<nav class="tab-bar">
+  <section class="left-small">
+    <a ng-click="cancel()" class="p10">
+      <span class="text-close" translate>Close</span>
+    </a>
+  </section>
+  <section class="middle tab-bar-section">
+    <h1 class="title ellipsis" ng-style="{'color':color}" translate>
+      Receive a customized amount
+    </h1>
+  </section>
+</nav>
+
+<div class="modal-content">
+  <div ng-show="customizedAmount">
+  <h4 class="title m0" translate>QR Code</h4>
+  <ul class="no-bullet size-14 m0">
+    <li class="line-b p10 oh text-center">
+      <qrcode size="220" data="bitcoin:{{addr + '?amount=' + customizedAmount}}"></qrcode>
+    </li>
+  </ul>
+  <h4 class="title m0" translate>Details</h4>
+  <ul class="no-bullet size-14 m0">
+    <li class="line-b p10 oh">
+    <span class="text-gray" translate>Address</span>:
+    <span class="right">
+      <span class="text-gray enable_text_select">{{addr}}</span>
+    </span>
+    </li>
+    <li class="line-b p10 oh">
+    <span class="text-gray" translate>Amount</span>:
+    <span class="right">
+      {{customizedAmount}}
+      <span class="label gray radius">{{customizedAlternative}}</span>
+    </span>
+    </li>
+  </ul>
+  </div>
+  <div ng-show="!customizedAmount" class="row m20t">
+    <div class="large-12 large-centered columns">
+      <form name="amountForm" ng-submit="submitForm(amountForm)" novalidate>
+        <div class="right" ng-hide="amountForm.amount.$pristine && !amountForm.amount.$modelValue ">
+          <span class="has-error right size-12" ng-if="amountForm.amount.$invalid">
+            <i class="icon-close-circle size-14"></i>
+            <span clas="vm" translate>Not valid</span>
+          </span>
+          <small class="text-primary right" ng-if="!amountForm.amount.$invalid">
+            <i class="icon-checkmark-circle size-14"></i>
+          </small>
+        </div>
+        <div ng-if="!showAlternative">
+          <label for="amount">
+            <span translate>Amount</span>
+          </label>
+          <div class="input">
+            <input type="number" id="amount" name="amount" ng-attr-placeholder="{{'Amount'|translate}}"
+            ng-minlength="0.00000001" ng-maxlength="10000000000" ng-model="_customAmount" valid-amount required autocomplete="off">
```

Comment:
I think `ng-minlength` & `ng-maxlength` are used to check string length. Are you looking for `ng-min` & `ng-max`? Also, the max value is 100 BTC when unit is set to bits. Is this a reasonable limit?